nft-devnet-0.vars

```diff
@@ -28,6 +28,8 @@ RETH_IMAGE=ethpandaops/reth:main
 
 LODESTAR_IMAGE=chainsafe/lodestar:next
 
+LODESTAR_HEAP=16384
```

Comment:
this should not be required to be higher in nft testnets since we fixed the issue that caused the memory increase